Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -193,20 +221,33 @@ extension ServiceRunnable {
     private func createRestRequest<T: Encodable>(
         path: PathType,
         method: RestMethod,
-        headers: [HTTPHeader]? = nil,
+        headers: [HTTPHeader],
         queryParameters: [HTTPQueryParameters]? = nil,
         body: T
     ) async throws -> RestRequest {
         let path = String(describing: path)
-        guard let url = baseURL?.appendingPathComponent(path) else {
+        guard var url = baseURL?.appendingPathComponent(path) else {
             throw URLError(.badURL)
         }
 
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys, .withoutEscapingSlashes]
+        let payload = try encoder.encode(body)
+        guard let header = headers.first(where: { $0.name == "SmileID-Request-Timestamp" }) else {
+            throw SmileIDError.missingHeader("Header with name `SmileId-Request-Timestamp` not found.")
```

Comment:
Do we want to prevent job submission if the timestamp is missing or leave that to the backend?